### DISCUSSION TITLE
Is it possible to create a clone an existing instance?

### DISCUSSION THREAD

**Discussion by Mashiane** [pockethost/pockethost#416] [Q&A]
Hi

Is there a way to create a clone of an existing instance. For example, i have collections with data that I will like to re-use. The instance can act as a **template** that I can just use when ever I have to deploy a similar app Ive developed.

Is there a way to just create a clone of an instance?

Thanks.

**Comment by benallfree** [comment]
Interesting question. This should work as long as you aren't using S3 storage:

1. Back up your data via the PocketBase admin
2. Download data
3. Create a new PocketHost instance
4. Restore the backup
